quoridor-app/app/api/gameApi.js

```diff
@@ -31,15 +31,15 @@ export const joinGame = gameId => {
   });
 };
 
-export const movePawn = (authToken, gameId, position) => {
-  return fetch(`${API_BASE_URL}/games/${gameId}/move-pawn`, {
+export const movePawn = (authToken, move) => {
```

Comment:
It could have been: ` export const movePawn = (authToken, { gameId, position })`.